SparkPost/2web2ui
# FE-876: Matchbox label component wrapper

### What Changed
  - Label component wrapper for hibana

### How To Test
 - enable hibana ui flag
```json
{ "options": { "ui":{ "hibana":{ "hasThemeControls": true } } } }
```
 - Verify places using Label component don't have visual issues with and without hibana enabled
